src/core/Akka.Remote/RemoteSystemDaemon.cs

```diff
@@ -291,28 +294,18 @@ public override IActorRef GetChild(IEnumerable<string> name)
 
         private IInternalActorRef GetChild(string name)
         {
+            var nameAndUid = ActorCell.SplitNameAndUid(name);
             IInternalActorRef child;
-            if (TryGetChild(name, out child))
+            if (TryGetChild(nameAndUid.Name, out child))
```

Comment:
Split the name and UID before we attempt to lookup the remotely deployed actors.